packages/mobile/src/firebase/firebase.ts

```diff
@@ -114,13 +114,45 @@ export const writePaymentRequest = (paymentInfo: PaymentRequest) => async () =>
   }
 }
 
+function isDeprecatedVersion(version: string, minVersion: string): boolean {
+  const minVersionArray = minVersion.split('.')
+  const versionArray = version.split('.')
+  const minVersionLength = Math.min(minVersionArray.length, version.length)
+  for (let i = 0; i < minVersionLength; i++) {
+    if (minVersionArray[i] > versionArray[i]) {
+      return true
+    } else if (minVersionArray[i] < versionArray[i]) {
+      return false
+    }
+  }
+  if (minVersionArray.length > versionArray.length) {
+    return true
+  }
+  return false
+}
+
 export async function getVersionInfo(version: string) {
+  // backward compatible
+  let deprecated = false
   const versionFSPath = version.split('.').join('/')
   Logger.info(TAG, `Checking version info ${version}`)
-  return (await firebase
+  const versionInfo = (await firebase
     .database()
     .ref(`versions/${versionFSPath}`)
     .once('value')).val()
+  if (versionInfo && versionInfo.deprecated !== undefined) {
```

Comment:
Can you add a comment above this function about explaining the expected format for the version in in the db